plugins/org.eclipse.thym.ui/src/org/eclipse/thym/ui/internal/preferences/EnginePreferencesPage.java

```diff
@@ -108,7 +108,11 @@ public boolean performOk() {
 				HybridMobileEngine engine =(HybridMobileEngine) selections[i];
 				prefVal.append(engine.getId());
 				prefVal.append(":");
-				prefVal.append(engine.getVersion());
+				if (engine.isManaged()) {
```

Comment:
Previously, creating a new Hybrid Mobile project with an engine added from a directory would cause an error, since the platform-spec written to config.xml was incorrect.